test/test_kron.jl

```diff
@@ -5,13 +5,14 @@ for A in Any[rand(2, 3), rand(-3:3, 2, 3),
                rand(2, 3) + im * rand(2, 3),
                sprand(10, 10, 0.1)]
     K = kron(A, B)
+    normK = norm(K, 1)
     T1 = kron(LinearOperator(A), B)
     T2 = kron(A, LinearOperator(B))
     T3 = kron(LinearOperator(A), LinearOperator(B))
     for T in [T1, T2, T3]
-      @test K == full(T)
-      @test K' == full(T')
-      @test K.' == full(T.')
+      @test norm(K - full(T)) < eps() * normK
+      @test norm(K' - full(T')) < eps() * normK
+      @test norm(K.' - full(T.')) < eps() * normK
```

Comment:
Sorry to be annoying but it should be `norm(..., 1)` on the lhs too.

Reply:
Sorry about that. Updated.